src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -118,6 +131,7 @@ interface INodeLibraryDependencyInfo
         /// <summary>
         /// Indicates whether this dependency is loaded in the current session
         /// </summary>
+        [Obsolete("This property is obsolete", false)]
```

Comment:
Why can't we just remove this property? These are new, unreleased interfaces and types, right?

Reply:
They are already in 2.3 branch, and cannot be removed form 2.3 - @reddyashish mentioned the conflict above.

Reply:
I'm sorry I still don't understand, maybe I'm missing something. 2.3 hasn't been released, has it? You mean daily builds have gone out?

Reply:
@aparajit-pratap let me try to clarify.
1. If we remove this API from here - we need to remove it from 2.3 - thats fine, as you said 2.3 is not released.
2. We cannot remove it from 2.3 because the work that @reddyashish did in this PR to replace it is not yet present in that branch.

so - we can either obsolete it. OR cherry pick this entire PR back to 2.3.
I was hoping we could stop working on 2.3 and start testing it. Does this make sense, what do you think?

Reply:
Sorry, I was assuming this PR is going into 2.3 as well. Thanks for the clarification.